### DISCUSSION TITLE
Issue with downloading workflow artifacts.

### DISCUSSION THREAD

**Discussion by michaljrk** [community/community#37230] [Actions]
I noticed today that downloading GitHub Actions workflow artifacts is broken (the browser progress bar is moving, but never gets to download the file, I tried from different browsers, private modes, etc. nothing helps).

The same happens for all artifacts, even from builds executed few days/weeks ago (that were downloadable just yesterday). I checked my quota/budget limits, retention policy and there is nothing that would indicate to be the cause.

**Comment by airtower-luna** [comment]
I've seen others mention similar problems, so it's probably a bug. You might want to [send a bug report](https://support.github.com/contact/bug-report) to GitHub support, to make sure they're aware of the problem whether they check the forum or not.
